app/lib/ui/components/template/component_detail_screen.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:ods_flutter/guidelines/spacings.dart';
+import 'package:ods_flutter_demo/ui/components/template/component.dart';
+import 'package:ods_flutter_demo/ui/utilities/display_image.dart';
+
+class ComponentDetailScreen extends StatelessWidget {
+  final OdsComponent component;
+
+  const ComponentDetailScreen({required this.component});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: Text(component.title),
+        ),
+        body: Column(
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+          children: [
+            displayImage(
+              component.imageResourceName,
+              MediaQuery.of(context).size.width,
+            ),
+            Padding(
+              padding: const EdgeInsets.only(
+                  top: spacingM,
+                  right: spacingM,
+                  left: spacingM,
+                  bottom: spacingM),
```

Comment:
When the padding is the same everywhere, prefer the use of `EdgeInsets.all(spacingM)`